html/html-forms/html-forms_Finding-a11y-errors/index.html

```diff
@@ -17,15 +17,22 @@ <h1 class="title">Understanding A11y in Forms</h1>
       </p>
 
       <form for="title">
-        <fieldset>
+        <fieldset for="description">
           <legend>Enter your information</legend>
-          <label aria-describedby="name-input">Name:</label>
+          <label for="name-Input" aria-describedby="name-input">Name:</label>
```

Comment:
I'm not sure whether the aria-describedby should be kept here or not. Let's check with Sven.